enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/context/InteractiveQuestionProvider.tsx

```diff
@@ -37,7 +37,7 @@ export const InteractiveQuestionContext = createContext<
   InteractiveQuestionContextType | undefined
 >(undefined);
 
-type InteractiveQuestionProviderProps = PropsWithChildren<
+export type InteractiveQuestionProviderProps = PropsWithChildren<
```

Comment:
this isn't used. I, however, see another declaration with the same type name in a different file
https://github.com/metabase/metabase/blob/12a0a334d7393c84768d7f28c376e2d54ad0c899/enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/context/InteractiveQuestionProviderWithLocation.tsx#L11

Reply:
consolidated the types into a `context/types` file